app/controllers/workspaces_controller.rb

```diff
@@ -40,5 +40,6 @@ def workspace_params
 
```

Comment:
![image](https://user-images.githubusercontent.com/55313319/68709345-cd3fd500-0595-11ea-85b3-4a75bb92268c.png)

Is authorize@workspace only in the set_workspace methode and is that only because the controller is not finished yet? or do i get it wrong?

Reply:
This screenshot is from the github project from todays lecture https://github.com/patricknpinto/restaurants_with_pundit/commit/00264750efdbb061c566ce40bd3d0acd5e95f1f1

Reply:
naming in the controller is wrong , should be set_booking

Reply:
the authorize will be in other methods but i havent gotten to that stage yet

Reply:
Ahh ok alright